src/js/video.js

```diff
@@ -389,6 +402,32 @@ videojs.createTimeRange = createTimeRange;
  */
 videojs.xhr = xhr;
 
+/**
+ * Resolve and parse the elements of a URL
+ *
+ * @param  {String} url The url to parse
+ * @return {Object}     An object of url details
+ * @method parseUrl
+ */
+videojs.parseUrl = Url.parseUrl;
+
+/**
+ * Should round off a number to a decimal place
+ *
+ * @param  {Number} num Number to round
+ * @param  {Number} dec Number of decimal places to round to
+ * @return {Number}     Rounded number
+ * @method roundFloat
+ */
+videojs.round = roundFloat;
+
+/**
+ * Event target class.
+ *
+ * @type {Function}
+ */
+videojs.EventTarget = EventEmitter;
```

Comment:
if we are changing `EventEmitter` to `EventTarget`, I'd prefer to just rename it internally as well and not just on export.